src/main/java/com/softserve/paymentservice/model/ErrorResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.softserve.paymentservice.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@Builder
+public class ErrorResponse {
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd hh:mm:ss")
+    private LocalDateTime time;
```

Comment:
use Instant instead

Reply:
remove this class, please) Spring has already generic exceptions